deploy/cluster-manager/config/samples/operator_open-cluster-management_clustermanagers.cr.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: operator.open-cluster-management.io/v1
 kind: ClusterManager
 metadata:
-  name: cluster-manager
+  name: cluster-manager-name
```

Comment:
why change this name?

Reply:
This is because, in the makefile, we use sed to replace this field with the env var we set. 
If here the name is "cluster-manager", it will also affect other fields.

Reply:
Done. Removed this.